mono/utils/atomic.h

```diff
@@ -464,4 +464,12 @@ extern void InterlockedWritePointer(volatile gpointer *dst, gpointer val);
 #define InterlockedAddP(p,add) InterlockedAdd64 ((volatile gint64*)p, (gint64)add)
 #endif
 
+/* The following functions cannot be found on any platform, and thus they can be declared without further existence checks */
+
+static inline gint32
+InterlockedSubtract (volatile gint32 *dest, gint32 sub)
+{
```

Comment:
I would say it doesn't matter for now, since it's a fairly rare occurrence for us to change a racy operation to an atomic one or vice versa. Also, whenever I get around to refactoring the atomics API, I'll update both APIs to be similar.